Main.java

```diff
@@ -9,7 +11,29 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    //create a Scanner for user input
+   Scanner input = new Scanner(System.in);
+  
+   //ask user for integer
+   System.out.println("Enter an integer");
+
+   //get integer from user
+   int integer = input.nextInt();
+
+   //get the remaider
+   int remainder = integer % 2;
+
+   //determine if integer is even or odd
+   if(remainder > 2){
+
+   //tell them they picked an odd number 
+   System.out.println("The integer you chose was odd");
```

Comment:
Watch your indentation here. If this is inside of an if statement, it should have an extra TAB over. Try hitting the auto-format button just before you are ready to submit to see if it makes some adjustments for you.